src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const elements = document.querySelectorAll('span.population');
```

Comment:
Consider checking if the elements are found before proceeding with calculations to avoid potential errors if the elements are not present in the DOM.